sirepo/html.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this file be named srhtml? When do you add the sr prefix (ex srtime?).

Reply:
I thought about srhtml, but realized we want it in all cases except for the few jinja files that are loaded before boot. I will address this more in response to https://github.com/radiasoft/sirepo/pull/2695#pullrequestreview-455890322

The sr prefix is a mistake as is the pk prefix except in rare instances like pkdlog and PKDict, which we want to use "everywhere". If we just always used fully qualified names, sirepo.time and sirepo.db, we wouldn't have any problems. It was a bit a problem in py2, but we had absolute_imports so it wasn't a problem.